### ISSUE TITLE
Use training (past time points) series as input when evaluating the model

### ISSUE THREAD

**Issue by ben-jy** [ontime-re/ontime#93]
Currently, only test series is used to evaluate the model, even for the input. However, as soon as no past values are used as "label", and that no futur data (i.e. test set) is used during training, there is no data leakage.
Using training data as input for evaluation increase the number of windows, and ensure that all test data is compared to predictions.
